### PR TITLE
Remove JMeter based performance tests

### PR DESCRIPTION
**Pull request by steven-sheehy**
**Detailed description**:
- Fix acceptance test config overwriting default config
- Fix monitor config documentation not showing `hedera.mirror.monitor` prefix and confusing users
- Fix test image sometimes failing to run due to not downloading all dependencies during build
- Improve test image creation time and image size (1.5GB to 566MB)
- Remove all JMeter related code

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
Follow up to #1940 

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Looks good.
Need to update `hedera-mirror-test/README.md` to remove/update "Performance Testing" section

File: hedera-mirror-test/README.md [hunk 1]
```diff
@@ -1,68 +1,100 @@
...
+allow us to confirm functionality on core scenarios that spanned the main and mirror networks. HCS specifically is a key
+scenario where transactions are submitted to the main network, the mirror node importer ingests these to the database,
+and the client subscribes to the mirror node gRCP API to receives messages in consensus order.
 
-To achieve this the tests utilize the Hedera Java SDK under the hood - https://github.com/hashgraph/hedera-sdk-java
+HTS is another key scenario where transactions are submitted to the main network, the mirror node importer ingests these
+to the DB, and the mirror node REST API is periodically invoked to verify transactions. This E2E suite gives us the
+ability to execute scenarios as regular clients would and gain the required confidence during deployment.
+
+To achieve this the tests utilize the [Hedera Java SDK](https://github.com/hashgraph/hedera-sdk-java) under the hood.
```
Comment (Nana-EC) [inline hedera-mirror-test/README.md:16]: nit: missing Scheduled scenario as a key scenario.
Can either add it or we can simple start listing the services and scenario since the ingestion and checking with REST API's and network calls is the same approach for all

File: hedera-mirror-test/README.md [hunk 2]
```diff
@@ -82,265 +114,50 @@ hedera:
...
--   Ensure desired JMX file(s) are under `hedera-mirror-test/src/test/jmeter/`
--   Startup database and module, either through Docker Compose or through Spring Boot:
+Tags: Tags allow you to filter which cucumber scenarios and files are run. By default tests marked with the @Sanity tag
+are run. To run a different set of files different tags can be specified
 
-    `'docker-compose [-f composefile.yml] up' or '../mvnw spring-boot:run'`
+Acceptance test cases
 
--   Start the tests:
+`./mvnw clean integration-test --projects hedera-mirror-test/ -P=acceptance-tests -Dcucumber.filter.tags="@Acceptance"`
```
Comment (Nana-EC) [inline hedera-mirror-test/README.md:123]: we'll remove the upper case tags soon
```suggestion
`./mvnw clean integration-test --projects hedera-mirror-test/ -P=acceptance-tests -Dcucumber.filter.tags="@acceptance"`
```

File: hedera-mirror-test/README.md [hunk 3]
```diff
@@ -82,265 +114,50 @@ hedera:
...
 
-    `./mvnw clean integration-test -P=performance-tests --projects hedera-mirror-test`
+All cases
 
+`./mvnw clean integration-test --projects hedera-mirror-test/ -P=acceptance-tests -Dcucumber.filter.tags="@FullSuite"`
```
Comment (Nana-EC) [inline hedera-mirror-test/README.md:127]:
```suggestion
`./mvnw clean integration-test --projects hedera-mirror-test/ -P=acceptance-tests -Dcucumber.filter.tags="@fullsuite"`
```

File: hedera-mirror-test/README.md [hunk 4]
```diff
@@ -82,265 +114,50 @@ hedera:
...
 
+Negative cases
 
-## Test Configuration
+`./mvnw clean integration-test --projects hedera-mirror-test/ -P=acceptance-tests -Dcucumber.filter.tags="@Negative"`
```
Comment (Nana-EC) [inline hedera-mirror-test/README.md:131]:
```suggestion
`./mvnw clean integration-test --projects hedera-mirror-test/ -P=acceptance-tests -Dcucumber.filter.tags="@negative"`
```

File: hedera-mirror-test/README.md [hunk 5]
```diff
@@ -82,265 +114,50 @@ hedera:
...
 
-Using the initial basic test plan, load tests can be configured to achieve many number of scenarios concerned with historical, future messages and their combination within a given topic ID.
-Historical messages are populated, and incoming future messages are simulated over time whiles a connection is established to the gRPC server to subscribe to a topic.
-A specified number of threads per client perform a subscription to HCS and messages are observed and verified for validity of order.
-At the end the db is cleared to ensure each test can run independently and the db state is not altered.
+Edge cases
 
-For Subscribe Only tests no db operations are performed.
+`./mvnw clean integration-test --projects hedera-mirror-test/ -P=acceptance-tests -Dcucumber.filter.tags="@Edge"`
```
Comment (Nana-EC) [inline hedera-mirror-test/README.md:135]:
```suggestion
`./mvnw clean integration-test --projects hedera-mirror-test/ -P=acceptance-tests -Dcucumber.filter.tags="@edge"`
```

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Looks good, some suggestions on the README

File: hedera-mirror-test/README.md (hunk 1 above)
Comment (steven-sheehy) [reply hedera-mirror-test/README.md:13]: To keep from having to maintain an exhaustive list of acceptance test scenarios I've made this more generic and do not mention HCS or HTS.

File: hedera-mirror-test/README.md (hunk 2 above)
Comment (steven-sheehy) [reply hedera-mirror-test/README.md]: Changed all tags in README to lowercase

File: hedera-mirror-test/README.md [hunk 6]
```diff
@@ -1,68 +1,99 @@
...
 
-A BDD approach was desired for our E2E test strategy as it would ensure we more closely tracked valid customer scenarios.
-Cucumber is one framework that provides tools to follow this methodology. One benefit being that tests can be written in human readable text. This allows developers, PM's and designers to formulate tests that have connected code to run valid customer scenarios.
-Cucumber uses the Gherkin plain language parser to describe tests.
-Further details may be explored at https://cucumber.io/. Additionally, cucumbers BDD approach is explained here https://cucumber.io/docs/bdd/
+A [BDD](https://cucumber.io/docs/bdd/) approach is desired for our E2E test strategy as it ensures we closely track
+valid customer scenarios. Cucumber is one framework that provides tools to follow this methodology. One benefit being
+that tests can be written in human-readable text. This allows developers, project managers and quality assurance to
+formulate tests that are connected to code to run valid customer scenarios. [Cucumber](https://cucumber.io) uses the
+Gherkin plain language parser to describe tests.
```
Comment (ijungmann) [inline hedera-mirror-test/README.md:23]: Just an idea, this section reads as a bit choppy, I realize it's been that way:
```suggestion
Our E2E tests use the [Cucumber](https://cucumber.io) framework, which allows them to be written in a [BDD](https://cucumber.io/docs/bdd/) approach that ensures we closely track valid customer scenarios.  The framework allows tests to be written in a human-readable text format by way of the Gherkin plain language parser, which gives developers, project managers, and quality assurance the ability to write easy-to-read scenarios that connect to more complex code underneath.
```

File: hedera-mirror-test/README.md [hunk 7]
```diff
@@ -82,265 +113,50 @@ hedera:
...
-
--   Save test JMX file to `hedera-mirror-test/src/test/jmeter`
-
-## Performance Test Execution
-
--   Ensure desired JMX file(s) are under `hedera-mirror-test/src/test/jmeter/`
--   Startup database and module, either through Docker Compose or through Spring Boot:
 
-    `'docker-compose [-f composefile.yml] up' or '../mvnw spring-boot:run'`
+Tags: Tags allow you to filter which cucumber scenarios and files are run. By default tests marked with the @sanity tag
```
Comment (ijungmann) [inline hedera-mirror-test/README.md:117]: nit:
```suggestion
Tags: Tags allow you to filter which Cucumber scenarios and files are run. By default tests marked with the @sanity tag
```

**Review by Nana-EC — DISMISSED** [review]
LGTM

**Review by ijungmann — DISMISSED** [review]
A couple of nits in the README, just grammar stuff, but LGTM

File: hedera-mirror-test/README.md [hunk 8]
```diff
@@ -1,68 +1,100 @@
...
+This module covers the end-to-end (E2E) testing strategy employed by the mirror node for key scenarios.
 
 ## Overview
 
-In an effort to quickly confirm product capability during deployment windows, we desired to have E2E tests that would allow us to confirm functionality on core scenarios that spanned the main and mirror networks interactions.
-HCS specifically is a key scenario where transactions are submitted to the main network, the mirror node parser ingests these to the DB and the mirror node GRCP endpoint is subscribed to to obtain messages verifying transactions.
-HTS is another key scenerio where transactions are submitted to the main network, the mirror node parser ingests these to the DB, and the mirror node REST transaction endpoint is periodically hit to verify transactions.
-This E2E suite gives us the ability to execute scenarios as external users would and gain the required confidence during development cycles.
+In an effort to quickly confirm product capability during deployment windows, we desire to have E2E tests that would
+allow us to confirm functionality on core scenarios that spanned the main and mirror networks. HCS specifically is a key
```
[the remark this answers is not in the document]
Comment (steven-sheehy) [reply hedera-mirror-test/README.md]: Actually already had that fixed. You must've reviewed a stale copy.

File: hedera-mirror-test/README.md [hunk 9]
```diff
@@ -1,68 +1,100 @@
...
-This section covers the E2E testing strategy employed by the mirror node for key scenarios
+This module covers the end-to-end (E2E) testing strategy employed by the mirror node for key scenarios.
 
 ## Overview
 
-In an effort to quickly confirm product capability during deployment windows, we desired to have E2E tests that would allow us to confirm functionality on core scenarios that spanned the main and mirror networks interactions.
-HCS specifically is a key scenario where transactions are submitted to the main network, the mirror node parser ingests these to the DB and the mirror node GRCP endpoint is subscribed to to obtain messages verifying transactions.
-HTS is another key scenerio where transactions are submitted to the main network, the mirror node parser ingests these to the DB, and the mirror node REST transaction endpoint is periodically hit to verify transactions.
-This E2E suite gives us the ability to execute scenarios as external users would and gain the required confidence during development cycles.
+In an effort to quickly confirm product capability during deployment windows, we desire to have E2E tests that would
```
[the remark this answers is not in the document]
Comment (steven-sheehy) [reply hedera-mirror-test/README.md]: Fixed

File: hedera-mirror-test/README.md (hunk 6 above)
Comment (steven-sheehy) [reply hedera-mirror-test/README.md]: Copied that

File: hedera-mirror-test/README.md (hunk 7 above)
Comment (steven-sheehy) [reply hedera-mirror-test/README.md]: Uppered it

**Review by Nana-EC — APPROVED** [review]
LGTM

File: hedera-mirror-test/README.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (ijungmann) [reply hedera-mirror-test/README.md]: Yep, forgot to remove that comment after it got updated.

**Review by ijungmann — APPROVED** [review]
